hydra/controllers/client.py

```diff
@@ -816,3 +816,54 @@ def disable_metrics(self):
 
         self.restart_service()
         self.app.log.info(f'Successfully disabled all metrics reporting.')
+
+    @ex(
+        arguments=[
+            (
+                ['-n', '--name'],
+                {
+                    'help': 'name of network for key backup',
+                    'action': 'store',
+                    'dest': 'name'
+                }
+            ),
+            (
+                ['-D', '--destroy'],
+                {
+                    'help': 'Use to overwrite existing backups',
+                    'action': 'store_true',
+                    'dest': 'destroy'
+                }
+            ),
```

Comment:
Some users may not want to store their backups in ~/.hydra. I think we should add a parameter that lets users override the default `destination` with their own - this will need to be added to the restore command as well. If the user-provided backup location doesn't exist, we shouldn't create that directory for them - it probably means they fat-fingered it.

Reply:
Done.